k8s-install/config.yaml

```diff
@@ -0,0 +1,34 @@
+# This ConfigMap can be used to configure a self-hosted Canal installation.
+# See `canal.yaml` for an example of a Canal deployment which uses
+# the config in this ConfigMap.
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: canal-config 
+  namespace: kube-system
+data:
+  # Configure this with the location of your etcd cluster.
+  # This must also be configured in the cni_network_config below.
+  etcd_endpoints: "http://127.0.0.1:2379"
+
+  # The interface used by flannel for host <-> host communication.
+  flannel_iface: "eth1"
```

Comment:
We need this for deployments with multiple interfaces where the default route doesn't point the right way (like in the vagrant guide I've been testing).